.github/workflows/zeek-nightly.yml

```diff
@@ -0,0 +1,31 @@
+name: Zeek nightly tests
+
+on:
+  schedule:
+    - cron: '* * * * *'
+
+jobs:
+  btest-linux:
+    name: Run test suite against Zeek nightly
+    runs-on: ubuntu-latest
+    container: zeek/zeek-dev:latest
```

Comment:
Removing the pre-commit testing seems appropriate since there are no commits in nightly testing. And, if we continuously monitor development, what changes could pop up in an LTS change that we didn't already encounter?

I don't feel strongly about this since these tests run quickly, just want to make sure I understand.

Reply:
In any case, a single workflow would introduce less copy-n-paste code which would need to be maintained. I personally would still prefer a single workflow.

Reply:
Whoops — also, the `* * * * *` was from testing and needs dialing down to daily runs.

Reply:
> a single workflow would introduce less copy-n-paste code which would need to be maintained.

Okay, I'll give it a shot.

Reply:
This seems to be fixed, resolving.